src/vs/workbench/contrib/tasks/common/tasks.ts

```diff
@@ -1201,7 +1201,8 @@ export const enum TaskSettingId {
 	QuickOpenDetail = 'task.quickOpen.detail',
 	QuickOpenSkip = 'task.quickOpen.skip',
 	QuickOpenShowAll = 'task.quickOpen.showAll',
-	AllowAutomaticTasks = 'task.allowAutomaticTasks'
+	AllowAutomaticTasks = 'task.allowAutomaticTasks',
+	Reconnection = 'task.experimentalReconnection'
```

Comment:
For other experimental settings, we've been using `experimental` more like namespace, so `task.experimental.reconnection`. Does that make sense here too?

Reply:
yep 👍🏼